### PR TITLE
Adding guest config module (draft). 

### PR DESCRIPTION
**Pull request by jodi-boone**
# Description
We are adding the Guest Configuration extension to all versions of windows, Linux, and the corresponding README files.
Total size: 101 mb, install time: ~ 50 seconds
**For new tools, please provide total size and installation time.**

#### Related issue:
Customer Request. 

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by jodi-boone** [comment]
@nitinbps adding GuestConfiguration module to windows and linux git actions as per customer request.

**Comment by al-cheb** [comment]
I don't think we should add such unpopular PowerShell module.

GuestConfiguration:
image


Az module:
image

File: images/linux/Ubuntu1804-Readme.md [hunk 1]
```diff
@@ -227,10 +227,11 @@
 - PowerShell 7.2.1
 
 #### PowerShell Modules
-| Module     | Version |
-| ---------- | ------- |
-| MarkdownPS | 1.9     |
-| Pester     | 5.3.1   |
+| Module               | Version |
```
Comment (al-cheb) [inline images/linux/Ubuntu1804-Readme.md:230]: Don't update auto-generated documentation.

**Comment by miketimofeev** [comment]
@jodi-boone I'm going to close this PR as no progression was made for 3 weeks and the module itself is not so popular. Feel free to contact us if you have any concerns.
Thank you!

**Pull request closed without merging**